test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -132,7 +132,7 @@ string TestFunctionCall::format(
 			bytes output = m_rawBytes;
 			bool const isFailure = m_failure;
 			result = isFailure ?
-				formatFailure(_errorReporter, m_call, output, _renderMode == RenderMode::ActualValuesExpectedGas, highlight) :
+				formatFailure(_errorReporter, m_call, output, _gasRenderMode == GasRenderMode::ExpectedGas, highlight) :
```

Comment:
I just noticed that this check was actually always true because we're in the `else` branch (so `if (_renderMode != RenderMode::ActualValuesExpectedGas)` turned out to be false). I think it should stay at true. It does not really have anything to do with gas.

```suggestion
				formatFailure(_errorReporter, m_call, output, /* _renderResult */ true, highlight) :
```